resolwe_bio/processes/import_data/etc.yml

```diff
@@ -7,8 +7,10 @@
   name: Expression time course
   requirements:
     expression-engine: jinja
+    resources:
+      network: true
```

Comment:
Check if the network resource is really needed as you are not using re-import here. Either support re-import or remove the network requirement.